### PR TITLE
feat: Have the option of not having a layout

### PR DESCRIPTION
**Pull request by flohhhh**
When screen is in "medium-screen" there is a margin top and a translate z.

The component must not have a defined layout,
we let the parent manage the display of the component.

We have the possibility of disable the layout.

### CODE REVIEW

File: react/Empty/index.jsx [hunk 1]
```diff
@@ -4,17 +4,36 @@ import PropTypes from 'prop-types'
...
   title,
   text,
   children,
   className,
+  layout,
   ...restProps
 }) => {
+  if (layout) {
+    depreciationLogger(
+      'The use of c-empty style is deprecated, Empty component must not have a defined layout'
```
Comment (ptbrowne) [inline react/Empty/index.jsx:22]: I think it'd be more explicit with "Please add layout=false in the props of the Empty component, and implement the vertical centering with a custom layout component".

File: react/Empty/index.jsx [hunk 2]
```diff
@@ -36,7 +55,12 @@ Empty.propTypes = {
   icon: iconPropType.isRequired,
   title: PropTypes.node.isRequired,
   text: PropTypes.node,
-  className: PropTypes.string
+  className: PropTypes.string,
+  layout: PropTypes.boolean
```
Comment (JF-Cozy) [inline react/Empty/index.jsx:59]: C'est normalement `PropTypes.bool`. Ca fonctionne avec `PropTypes.boolean` ?

Comment (flohhhh) [reply react/Empty/index.jsx:59]: Oops :)

File: react/Empty/index.jsx (hunk 1 above)
Comment (flohhhh) [reply react/Empty/index.jsx]: You're right

**Review by ptbrowne — APPROVED** [review]

**Pull request merged**